BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+
+   ![New Omni-Channel connection](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OmniChannelConnectionCreated.png "[New Omni-Channel connection]")
+
+
+## Configure BigCommerce connection 
+
+Configure the *BigCommerce* connection after you have created it.   
+
+In the *Omni-Channel* settings, you can configure the *Variant option type* that defines the way in which the offer variants are presented in the shop.   
+By the *Variant option type* setting, you can define how your customers can select a product variant in *BigCommerce*. Since *BigCommerce* allows multiple types for variant options, the driver needs a specification how to determine the preferred type for a variant option. You can define variant option types such as dropdown, radio button, or rectangle as default for your offers in the *Variant option type* setting.   
+For specific offer types, you can use the color-swatch type that is determined in a specific way. For detailed information, see [Configure color-swatch variant option](./04_ManageProductData.md#configure-color‐swatch-variant-option).
+
+With the *Order filter* setting, you can filter the orders that are to be imported from *BigCommerce* to *Omni-Channel*.  
+Additionally, if you have worked with *BigCommerce* before starting with the *Actindo Core1 Platform*, you can define the date or ID, from which existing orders should be imported.
+
+The *Offers* setting is only relevant for the first initial import of products from *BigCommerce*. You can specify that only those attributes are to be imported, that are mapped to the offer by the ETL data mapping. This allows you to maintain only certain attributes.
+
+
+#### Prerequisites
+
+- In the *Actindo Core1 Platform*, you are permitted to create connections in *Omni-Channel*.
+- A *BigCommerce* connection has been created.
+
+#### Procedure
+
+*Ommi-Channel > Settings > Tab CONNECTIONS*
+
+![Omni-Channel connections](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OmniChannelConnectionCreated.png "[Omni-Channel connections]")
+
+ 1. Click the *BigCommerce* connection you want to edit in the list of connections.   
+   The *Credentials* and the *Settings* tab are displayed in the *Edit connection* view. By default, the *Credentials* tab is selected.
+
+    ![Connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConnectionCredentials.png "[Connection credentials]")
+
+2. Click the *Settings* tab.
+
+3. Click the *Variant option type* entry.   
+   The *Variant option type used for variants* dropdown is displayed.
+
+    ![Variant option type](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionType.png "[Variant option type]")
+
+4. Select the *Variant option type* you want to use.
+   The following option types are available:
+   | Dropdown | Meaning   
```

Comment:
Zu den Layout: Ich finde die Tabelle an sich nicht schlecht, aber bei den "available options" nach einer Drop-down-Liste haben wir immer Aufzählung gemacht (also, bullet points in Markdown)...

Reply:
d